query-service-impl/src/main/java/org/hypertrace/core/query/service/postgres/PostgresClientFactory.java

```diff
@@ -131,5 +178,42 @@ static String resolveStatement(String query, Params params) {
     private static String getStringParam(String value) {
       return "'" + value.replace("'", "''") + "'";
     }
+
+    @SneakyThrows
+    Observable<Row> convert(ResultSet resultSet) {
+      List<Row.Builder> rowBuilderList = new ArrayList<>();
+      while (resultSet.next()) {
+        Row.Builder builder = Row.newBuilder();
+        rowBuilderList.add(builder);
```

Comment:
but if not, add at the end and create a list of rows rather than a list of row builders.

Reply:
Done